app/views/state_file/questions/az_public_school_contributions/_form.html.erb

```diff
@@ -9,79 +10,81 @@
     <% end %>
 
     <div class="question-with-follow-up">
-      <div class="question-with-follow-up__question">
-        <div class="white-group">
-          <%= f.cfa_radio_set(
-                :made_contribution,
-                label_text: t('state_file.questions.az_public_school_contributions.edit.made_contribution', count: current_intake.filer_count, year: current_tax_year),
-                collection: [
-                  { value: :yes, label: t('general.affirmative'), input_html: { "data-follow-up": "#contribution-details" } },
-                  { value: :no, label: t('general.negative') }
-                ]
-              )
-          %>
-        </div>
-      </div>
-
-      <div class="reveal">
-        <button class="reveal__button"><%= t('.which_qualify') %></button>
-        <div class="reveal__content">
-          <ul>
-            <% t(".qualifying_list").each do |list_item| %>
-              <li><%= list_item %></li>
+      <% if show_yes_no %>
+        <div class="question-with-follow-up__question">
+          <div class="white-group">
+            <%= f.fields_for(:state_file_az_intake) do |ff| %>
+              <%= ff.cfa_radio_set(
+                    :made_az322_contributions,
+                    label_text: t('.made_az322_contributions', count: current_intake.filer_count, year: current_tax_year),
+                    collection: [
+                      { value: :yes, label: t('general.affirmative'), input_html: { "data-follow-up": "#contribution-details", } },
+                      { value: :no, label: t('general.negative') }
+                    ]
+                  ) %>
             <% end %>
-          </ul>
-          <p><%= t(".more_details_html") %></p>
-        </div>
-      </div>
+          </div>
+          <div class="reveal">
+            <button class="reveal__button"><%= t('.which_qualify') %></button>
+            <div class="reveal__content">
+              <ul>
+                <% t(".qualifying_list").each do |list_item| %>
+                  <li><%= list_item %></li>
+                <% end %>
+              </ul>
+              <p><%= t(".more_details_html") %></p>
+            </div>
+          </div>
 
-      <div class="reveal">
-        <button class="reveal__button"><%= t(".donations_this_year", current_year: current_tax_year + 1) %></button>
-        <div class="reveal__content">
-          <p><%= t(".donations_this_year_details_html", current_year: current_tax_year + 1) %></p>
+          <div class="reveal">
+            <button class="reveal__button"><%= t(".donations_this_year", current_year: current_tax_year + 1) %></button>
+            <div class="reveal__content">
+              <p><%= t(".donations_this_year_details_html", current_year: current_tax_year + 1) %></p>
+            </div>
+          </div>
         </div>
-      </div>
+      <% end %>
+    </div>
```

Comment:
Shouldn't the `div.question-with-follow-up` contain both the `*__question` and `*__follow-up` elements? Maybe my diff viewer is messing this up, but it looks like you're closing the container div before the follow-up.

Reply:
I just noticed that on heroku selecting Yes on the initial/blank page does show the form fields, but then selecting No doesn't hide it again. That might be because of this.

Reply:
oops!